atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -212,16 +212,16 @@ boolean allowHiddenTableAccess() {
      *
      * @see TransactionManagers#createInMemory(Set)
      */
-    public static SerializableTransactionManager createInMemory(Schema schema) {
+    public static TransactionManager createInMemory(Schema schema) {
```

Comment:
you fancy deleting the no-args constructor of SerializableTransactionManager?

Reply:
This is currently used for `AutoDelegate_SerializableTM`, which is used once internally for the async init wrapper and once externally in a case where they want to change the `createTransaction` behavior but need access to the ~75 member variables. The second case can be switched to `AutoDelegate_TM` instead. I have to dig in some to see if we can fix the init wrapper usage.